src/shared/components/app-definitions.ts

```diff
@@ -387,6 +387,26 @@ const NEONMODEM: AppDetails = {
   platforms: [Platform.CLI],
 };
 
+const SUMMIT: AppDetails = {
+  name: "Summit",
+  description: "A Lemmy Client for Android.",
+  link: "https://play.google.com/store/apps/details?id=com.idunnololz.summit",
+  icon: "/static/assets/images/summit_logo.svg",
+  banner: "/static/assets/images/summit_screen.webp",
+  links: [
+    {
+      link: "https://play.google.com/store/apps/details?id=com.idunnololz.summit",
+      icon: "googleplay",
+    },
+    {
+      link: "https://github.com/idunnololz/summit-for-lemmy",
+      icon: "github",
+    },
+  ],
+  sourceType: SourceType.Closed,
```

Comment:
Should be `SourceType.Open`

Reply:
No its correct, the github repo doesnt contain any code.

Reply:
Ah I see.